tests/pos/i15784.scala

```diff
@@ -2,7 +2,4 @@ def i15784 = List(42) match
   case List(_, rest @ _*) => rest
   case List(_, Rest @ _*) => Rest
   case List(_, `Rest` @ _*) => Rest
-  case _ => ???
-
-def i15784_auxiliary = 42 match
-  case `type` : Int => `type`
+  case _ => ???
```

Comment:
You could quiet the other warnings and add the `-Werror` in the existing file. These directories get large.

By quiet, I'd suggest a def for each "rest" test. (and not just nowarn.)